migtests/scripts/functions.sh

```diff
@@ -30,28 +30,52 @@ print_env() {
 run_psql() {
 	db_name=$1
 	sql=$2
-	psql "postgresql://${SOURCE_DB_USER}:${SOURCE_DB_PASSWORD}@${SOURCE_DB_HOST}:${SOURCE_DB_PORT}/${db_name}" -c "${sql}"
+	psql "postgresql://${SOURCE_DB_ADMIN_USER}:${SOURCE_DB_ADMIN_PASSWORD}@${SOURCE_DB_HOST}:${SOURCE_DB_PORT}/${db_name}" -c "${sql}"
+}
+
+grant_user_permission_postgresql() {
+	db_name=$1
+
+	echo "SELECT 'GRANT USAGE ON SCHEMA ' || schema_name || ' TO ${SOURCE_DB_USER};' FROM information_schema.schemata; \gexec" | psql "postgresql://postgres:secret@${SOURCE_DB_HOST}:${SOURCE_DB_PORT}/${db_name}" 
```

Comment:
Simplify this and similar other statements by splitting them:
```
conn_string="..."
cmd="SELECT ..."
echo "${cmd}" | psql ${conn_string}
```
Never pack too much into one statement. The intent of the code should be easily visible.

Once you implement above suggestion, you will need to define `conn_string` only once. Also, use the `SOURCE_DB_ADMIN_USER` and `SOURCE_DB_ADMIN_PASSWORD` in the `conn_string`.